recipes/wcsaxes/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.7" %}
+
+package:
+  name: wcsaxes
+  version: {{version}}
+
+source:
+  fn: wcsaxes-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/w/wcsaxes/wcsaxes-{{ version }}.tar.gz 
+  md5: 926ca8579b05667f1c9b6ca807645bc7
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >1.7.1
+    - astropy >=1.0
+    - matplotlib
+
+  run:
+    - python
+    - numpy >1.7.1
+    - astropy >=1.0
+    - matplotlib
+
+test:
+  # Python imports
+  imports:
+    - wcsaxes
```

Comment:
You could technically also run the test suite with ``wcsaxes.test()``, it will skip the image tests anyway